modules/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/parser/BLangPackageBuilder.java

```diff
@@ -289,7 +294,7 @@ private void addExpressionNode(ExpressionNode expressionNode) {
     public void addLiteralValue(Object value) {
         LiteralNode litExpr = TreeBuilder.createLiteralExpression();
         litExpr.setValue(value);
-        addExpressionNode(litExpr);
+        this.exprNodeStack.push(litExpr);
```

Comment:
Why is this line changed? .. same change below also ..

Reply:
Reverted to original. This was changed to fix issue with addExpressionNode method which was there previously. It was fixed by #3286 already and hence reverted to the original code. 
Thanks.